packages/assets/lib/returnImage.js

```diff
@@ -105,12 +119,6 @@ module.exports = async ({
           quality: 80
         })
       }
-
-      // update 'Content-Type'
-      res.set('Content-Type', webp
-        ? 'image/webp'
-        : mime
-      )
```

Comment:
did you check if Content-Type gets set automatically to mime? This is not set explicitly anymore.

Reply:
as far as i understand it was redundant. There is already a set above if mime is detected: https://github.com/orbiting/backends/pull/29/files#diff-3dbdb44a7074aa4e3203947972d99e46R85

Reply:
👍